### ISSUE TITLE
Grid Features Button Appearing In Toolbar but Not Working

### ISSUE THREAD

**Issue by Lisabeth** [unosquare/tubular-react#959]
**Describe the bug**
Under certain circumstances, the "grid features" button is visible, but clicking on it has no effect.

**To Reproduce**
1. Go to [this Codesandbox](https://codesandbox.io/s/holy-meadow-rebot?file=/src/App.js)
2. Click on the "grid features" button in the top right corner, to the right of the search text area
3. Nothing happens

**Expected behavior**
When I have no filterable columns, the grid features button should not appear. I searched through ToolbarOptions but this button is not included in the list of features to turn on and off that way. I've seen several examples online where the grid features button does not appear, but it is not clear why that is happening in these examples.

**Screenshots**
n/a, see codesandbox

**Desktop (please complete the following information):**
 - OS: latest MacOS
 - Browser: Chrome
 - Version: latest Chrome

**Additional context**

**Comment by bmitchinson** [comment]
Also looking to remove these controls but cannot determine how

**Comment by bmitchinson** [comment]
Was hoping that there was a conditional on this line: https://github.com/unosquare/tubular-react/blob/598b28ff1166fce323fbe8b4faec057f25671031/src/Toolbar/GridToolbar.tsx#L99

As it renders the following

```html
<button class="MuiButtonBase-root MuiIconButton-root" tabindex="0" type="button" aria-label="Grid features" title="Grid features">
  <span class="MuiIconButton-label">
      <svg class="MuiSvgIcon-root" focusable="false" viewBox="0 0 24 24" aria-hidden="true">
        <path d={...}></path>
      </svg>
  </span>
  <span class="MuiTouchRipple-root">
  </span>
</button>
```

@Lisabeth can you link an example where this button doesn't appear?
Setting the display attribute of this element to `none` achieves what I need it to, but it doesn't even have an id / unique class for me to hide it.

**Comment by kadosh** [comment]
Hi @bmitchinson,

I'm sorry for the late response but I've been quite busy. We don't have a way at this moment to hide that button. The workaround is that you set at least one of your string columns as filterable: true, so that it can show you some filter functionality.

I will do my best to fix this ASAP.

**Comment by kadosh** [comment]
Filters button will be hidden when no filterable columns present. Fixed in 4.2.11

cc @Lisabeth , @bmitchinson
